PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -360,6 +449,13 @@ struct HfTaskFlowCharmHadrons {
     if (cent < centralityMin || cent > centralityMax) {
       return;
     }
+    float occupancy = 0.;
+    uint16_t hfevflag;
+    if (storeOccupancy) {
+      occupancy = getOccupancy(collision);
+      registry.fill(HIST("trackOccVsFT0COcc"), collision.trackOccupancyInTimeRange(), collision.ft0cOccupancyInTimeRange());
+      hfevflag = static_cast<int>(hfEvSel.getHfCollisionRejectionMask<true, o2::hf_centrality::CentralityEstimator::None, aod::BCsWithTimestamps>(collision, cent, ccdb, registry));
```

Comment:
`getHfCollisionRejectionMask` returns `uint16_t`. Why would you cast it to `int` and back?

Reply:
It was indeed not necessary. Thanks for noticing!